### PR TITLE
release 1.2.2

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -45,3 +45,10 @@ releases:
       bugfixes:
       - fix parsing of openssl certificate subject for EL7 and EL8
     release_date: '2022-05-23'
+  1.2.2:
+    changes:
+      bugfixes:
+      - cloud_connector role - use a regular expression to parse the CN from the cert
+    fragments:
+    - 94-subject-regex.yml
```
Comment (ehelms) [inline changelogs/changelog.yaml:53]: Something appears to have actually deleted the fragment in this PR.

Comment (evgeni) [reply changelogs/changelog.yaml:53]: Yes, which is fine. The fragment is only needed for the generation of the changelog, not afterwards.

Comment (evgeni) [reply changelogs/changelog.yaml:53]: This line says: we used that fragment.

As you can configure antsibull not to delete, but then you need to track which are already used.

Comment (ehelms) [reply changelogs/changelog.yaml:53]: Ohh... that's weird.

**Review by ehelms — APPROVED** [review]

**Pull request merged**